Server/src/models/AuthenticationModel.ts

```diff
@@ -127,4 +127,33 @@ export class AuthenticationModel {
 
         let connection = getConnection();
     }
+
+
+    //for issue 146
+    static async updateUserPasswordDetail(email: string, passwordHash: string): Promise<boolean> {
```

Comment:
I think this can be called just updateUserPassword, don't think we need detail here and the param can be called newPasswordHash for clarity

Reply:
@ratulkai93 This is still valid

Reply:
Will this be changed?

Reply:
i believe i fixed it on the recent commit